airflow/contrib/hooks/gcs_hook.py

```diff
@@ -253,8 +253,8 @@ def is_updated_after(self, bucket, object, ts):
 
             if blob_update_time > ts:
                 return True
-            else:
-                return False
+
+        return False
```

Comment:
👍

Reply:
Can you approve this PR if you are ok with it :) ?

Reply:
Sure! done. I like it :)